deploy/start_flower.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+cd ${PROJECT_NAME}
+until timeout 5s celery -A ${PROJECT_NAME} inspect ping; do
+    >&2 echo "Celery workers not available"
+done
+
+echo 'Starting flower'
+celery -A ${PROJECT_NAME} flower --port=5555 --broker=redis://redis:6379/0
```

Comment:
@deepakdinesh1123 
do we need to pass `broker` explicitly over here?
I think it will take the broker URL from `CELERY_BROKER_URL` settings variable